_sass/partials/ideasDone.scss

```diff
@@ -0,0 +1,20 @@
+.section-title{
+  margin-bottom : 100px;
+}
+
+.ideasContainer {
+  width : 90%;
+  display : flex;
+  flex-wrap: wrap;
+  margin-left : 5%;
+  justify-content: space-evenly;
+
+}
+.ideaItem {
```

Comment:
I would add a white background on the items, so that it's a bit easier to read
I'd change the padding to `24px 24px 6px 24px` to get the text away from the edge :)
Otherwise, perfect!